app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -29,6 +29,22 @@ def textual_group_smart_management
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_group_volumes
+    labels = [_("Name"), _("Property"), _("Value")]
+    h = {:labels => labels}
+    h[:values] = []
```

Comment:
Please shorten that with:

    h = {:labels => [_("Name"), _("Property"), _("Value")], :values => []}